ApplicationInsights/JavaScriptSDK.Interfaces/Context/ISample.ts

```diff
@@ -1,6 +0,0 @@
-﻿export interface ISample {
```

Comment:
Did you delete this file altogether?

Reply:
This is currently not being used. It'll eventually come back when we create a sampling  plugin. I also removed Sample.ts which is the class that implements ISample. A sample object was created in TelemetryContext. I'll bring it back so we don't lose track of it.

---
In reply to: [214086514](https://github.com/Microsoft/ApplicationInsights-JS/pull/660#discussion_r214086514) [](ancestors = 214086514)